_layouts/default.html

```diff
@@ -3,9 +3,36 @@
 <html lang="{{ page.lang | default: site.lang | default: "es" }}">
   {% include head.html %}
   <body>
-    {%- include header.html scrolly_nav=_posts -%}
-    {{ content }}
-    {% include footer.html %}
+    <nav class='nav navbar navbar-expand flex-row-reverse'>
+      {% include language_switcher.html %}
+
+      <ul class="navbar-nav ml-auto">
+        <li class="nav-item"><a href='/blog.html' class='nav-link'>Blog</a></li>
```

Comment:
@sauloperez as far as I can see, there is no link to the jobs page. Is this by design or is it a mistake?

```suggestion
        <li class="nav-item"><a href='/blog.html' class='nav-link'>{% t pages.navigation.blog %}</a></li>
        <li class="nav-item"><a href='/jobs.html' class='nav-link'>{% t pages.navigation.jobs %}</a></li>
```

(If you accept this suggestion you'll still need to update the i18n files)

Reply:
Yes, still not there. That comes with the next commit. So far, only refactoring.